debugger/src/debugger.rs

```diff
@@ -100,7 +100,7 @@ pub struct DebuggerOptions {
     pub(crate) cwd: Option<PathBuf>,
 
     /// Binary to debug as a path. Relative to `cwd`, or fully qualified.
-    #[clap(long, parse(from_os_str), conflicts_with("dap"))]
+    #[clap(long, parse(from_os_str), required(true), conflicts_with("dap"))]
```

Comment:
I think it is odd that this field is required but an Option. Is that intended?

Reply:
This is because of the `conflicts_with`, so it can only be mandatory if the `dap` flag is not present.